marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx

```diff
@@ -55,10 +55,18 @@ describe("Mapping Card component", () => {
       getByRole = renderResults.getByRole;
       queryAllByRole = renderResults.queryAllByRole;
       getByText = renderResults.getByText;
+      getByTestId = renderResults.getByTestId;
     });
 
     expect(getByRole("edit-mapping")).toBeInTheDocument();
+    fireEvent.mouseOver(getByRole('edit-mapping'));
+    await wait (() => expect(getByText('Edit')).toBeInTheDocument());
+    expect(getByTestId("Mapping1-stepDetails")).toBeInTheDocument();
+    fireEvent.mouseOver(getByTestId('Mapping1-stepDetails'));
+    await wait (() => expect(getByText('Step Details')).toBeInTheDocument());
     expect(getByRole("settings-mapping")).toBeInTheDocument();
+    fireEvent.mouseOver(getByRole('settings-mapping'));
+    await wait (() => expect(getByText('Settings')).toBeInTheDocument());
     expect(queryAllByRole('delete-mapping')).toHaveLength(0);
     expect(getByRole('disabled-delete-mapping')).toBeInTheDocument();
 
```

Comment:
Since we're checking it for all the other icons, can we check the tooltip for the disabled trash icon as well?